### PR TITLE
Feat/hide more ads

### PR DESCRIPTION
**Pull request by garnetred**
## Overview
This change hides newer ads on LinkedIn, as well as updates the popup.html file to include more info. 

## Background
Recently, I noticed that I was seeing more and more ads on LinkedIn despite having this extension enabled. 

I noticed that the code had been updated and there were newer types of advertisements, so I went ahead and updated the code. 

I also created a Buy Me a Coffee account and decided to link it so that people have a way of supporting my work (all of the extensions I create are free and don't have any ads).

I decided to indicate the version number in the popup window and also link to the codebase itself as well.  

## Testing
More detailed instructions are located in the [README](https://github.com/garnetred/hide-linkedin-ads/blob/main/README.md). 

In short, you can load the extension locally and then go to LinkedIn. You should not see any ads on the homepage. When you click on the icon for the extension, you should also see information about the version, a link to the github, and a link to donate. 

## Screenshot(s):
Screen Shot 2023-09-22 at 4 13 30 AM

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Other

## Checklist:

- [x] I have self-reviewed my code and added comments where relevant.
- [ ] I have updated any relevant documentation.

### CODE REVIEW

File: css/global.css [hunk 1]
```diff
@@ -2,3 +2,7 @@
 div.feed-shared-update-v2:has(div):has(div.update-components-actor):has(.app-aware-link):has(div.update-components-actor__meta):has(span.update-components-actor__sub-description):has(span.visually-hidden):has(span[aria-hidden="true"]):has(span):not(:has([type="globe-americas"])){
```
Comment (garnetred) [inline css/global.css:2]: I kept this because I think they're probably phasing out the old ads to replace them with the new - possibly because of this extension and others like it. But I haven't tested yet to see if the old ads are still there, so in the future I may remove this.

File: css/global.css [hunk 2]
```diff
@@ -2,3 +2,7 @@
...
   display: none !important;
 }
+
+div.feed-shared-update-v2:has(div):has(div.update-components-actor):has(.app-aware-link):has(div.update-components-actor__meta):has(span.update-components-actor__sub-description):not(:has(div.update-components-text-view)){
```
Comment (garnetred) [inline css/global.css:6]: New ads don't have the child div with the class `update-components-text-view`.

File: manifest.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "manifest_version": 3,
   "name": "Hide LinkedIn Ads",
-  "version": "0.0.1",
+  "version": "0.0.2",
```
Comment (garnetred) [inline manifest.json:4]: Moving on up!
